src/maker/config.rs

```diff
@@ -53,20 +41,14 @@ impl Default for MakerConfig {
         Self {
             port: 6102,
             rpc_port: 6103,
-            heart_beat_interval_secs: 3,
-            rpc_ping_interval_secs: 60,
-            directory_servers_refresh_interval_secs: 60 * 60 * 12, //12 Hours
-            idle_connection_timeout: 300,
             absolute_fee_sats: Amount::from_sat(1000),
             amount_relative_fee_ppb: Amount::from_sat(10_000_000),
             time_relative_fee_ppb: Amount::from_sat(100_000),
-            required_confirms: 1,
             min_contract_reaction_time: 48,
             min_size: 10_000,
             socks_port: 19050,
-            directory_server_onion_address: "directoryhiddenserviceaddress.onion:8080".to_string(),
-            directory_server_clearnet_address: "127.0.0.1:8080".to_string(),
-            fidelity_value: 5_000_000, // 5 million  sats
+            directory_server_address: "directoryhiddenserviceaddress.onion:8080".to_string(),
+            fidelity_value: 5_000_000, // 5 million sats
```

Comment:
General question. Is 5 mil sats a good default? 
cc @Shourya742

Reply:
Can be lowered but works with our test..so no issue as of now..